### ISSUE TITLE
Add Stack Related Important Questions.

### ISSUE THREAD

**Issue by Kushal-W** [Kushal-W/AlgoInC#3]
The code should be in c or c++.
Question should be commented in the top of the solution code.

**Comment by AnkushGoel251** [comment]
Yes I would like to work on this, please assign this to me. I will code in C++.

**Comment by aniket2003** [comment]
@Kushal-W Could you please assign me with this task as I would like to do it also I am efficient at competitive programming and have solved some good problems on Stack . I can bring some good Questions .

**Comment by mvaja13** [comment]
please assign this to me

**Comment by soham2312** [comment]
Please assign this to me
